woocommerce/woocommerce-gutenberg-products-block
# Update Products on Sale and Products Tag icons

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/65529337-3ee49680-def6-11e9-9ada-331592f05441.png)
![image](https://user-images.githubusercontent.com/3616980/65529489-810dd800-def6-11e9-8344-8c31f8fc6055.png)

### How to test the changes in this Pull Request:

1. Create a post and add a _Products on Sale_ and _Products Tag_ blocks.
2. Verify they have the new icons.

### Changelog

> _Products on Sale_ and _Products Tag_ blocks received new icons.
